app/models/concerns/course/lesson_plan/item_todo_concern.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+module Course::LessonPlan::ItemTodoConcern
+  extend ActiveSupport::Concern
+
+  included do
+    after_save do |item|
+      if item.try(:actable_type).try(:constantize).try(:has_todo?) && item.draft_changed?
```

Comment:
Do you mean that we delegate this `has_todo?` in `lesson_plan_item` to `actable_type`?